### ISSUE TITLE
enable multiline support in picker dropdowns when react-virtualized is used

### ISSUE THREAD

**Issue by iwan-uschka** [rsuite/rsuite#1104]
### What problem does this feature solve?

I am using an `InputPicker` where `groupBy` is set. Right now the dropdown contains 3 groups and 7 items in total. If all groups are expanded `react-virtualized` is used an all items are cropped to a one liner. If a collapse one group `react-virtualized` is not used anymore an all items are shown completely containing multiple lines of text.

It would be cool to control somehow the item layout when `react-virtualized` is used because the switch between cropped/not cropped is not helpful.

Unfortunately it's not sufficient to just adapt the CSS because `react-virtualized` is setting height/width/position of each item.

If i try to set `listProps` like

```
<RsFormControl
  data={data}
  block
  placeholder={placeholder}
  name={name}
  listProps={{ rowHeight: 100 }}
  groupBy={'dataTypeTitle'}
  valueKey="key"
  labelKey="code"
  accepter={RsInputPicker}
/>
```

to manipulate `react-virtualized` i get an error:

```
Warning: React does not recognize the `listProps` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `listprops` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
```

In the end it would be cool to enable variable row heights.

### What does the proposed API look like?

Don't know.

**Comment by simonguo** [comment]
If you don’t have a lot of data, you can disable virtualized, so you can use CSS to wrap.

```jsx
<InputPicker 
  virtualized={false} 
  menuStyle={{ width: 224 }}  
  className="custom-picker"
/>
```
CSS:

```css
.custom-picker .rs-picker-select-menu-item{
    word-wrap: break-word;
    word-break: normal;
}
```

**Comment by iwan-uschka** [comment]
Setting `virtualized={false}` is a functioning workaround. Thx @simonguo. Feel free to leave the ticket open if you think that this issue needs to be solved in the future.

**Comment by jcady** [comment]
@simonguo Just wanted to give this a bump. Using `virtualized` still makes multiline rows get cutoff. In my case, my dataset is large and I need to use virtualized for performance reasons.

Here's a reproduction: https://codesandbox.io/s/wild-voice-ry0q5

**Comment by SevenOutman** [comment]
Fixed in #1962
